### PR TITLE
Part II, Lesson 5

### LINKED ISSUE #16: Part II, Lesson 5
This lesson will introduce object-to-BG interactions. The ball must start colliding against the bricks in the background as well, bouncing and eliminating them. Like for OBJs, VRAM shall be accessed directly to perform collision checks.

This can be made buggy the first time around, e.g. always checking above the ball; the bug can then be demonstrated (collision misbehaving when the ball hits bricks from the side and/or top), and fixed. Again, it's important to show how to debug and iterate on the code.

### PR DESCRIPTION
**Pull request by eievui5**
Closes #16

### CODE REVIEW

**Comment by eievui5** [comment]
I'll leave this as a draft until #37 is merged

File: src/part2/bricks.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Up until this point our ball hasn't done anything but bounce around, but now we're going to make it destroy the bricks.
+
+Before we start, let's go over a new concept: constants.
+We've already used some constants, like `rLCDC` from `hardware.inc`, but we can also create our own for anything we want.
+Let's make three constants at the top of our file, representing the tile IDs of left bricks, right bricks, and blank tiles.
+```rgbasm,linenos,start={{#line_no_of "" ../../unbricked/bricks/main.asm:constants}}
+{{#include ../../unbricked/bricks/main.asm:constants}}
+```
+
```
Comment (ISSOtm) [inline src/part2/bricks.md:11]:
```suggestion

Constants are a kind of *symbol* (which is to say, "a thing with a name").
Writing a constant's name in an expression is equivalent to writing the number the constant is equal to, so `ld a, BRICK_LEFT` is the same as `ld a, $05`.
But I think we can all agree that the former is much clearer, right?

```

File: src/part2/bricks.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+Now we'll write a function that checks for and destroys bricks.
+Our bricks are two tiles wide, so when we hit one we'll have to remove the adjacent tile as well.
+If we hit the left side of a brick (represented by `BRICK_LEFT`), we need to remove it and the tile to the right (which should be the right side).
```
Comment (ISSOtm) [inline src/part2/bricks.md:14]:
```suggestion
If we hit the left side of a brick (represented by `BRICK_LEFT`), we need to remove it and the tile to its right (which should be the right side).
```

File: src/part2/bricks.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Alternatively, if we hit the right side we need to remove the left!
```
Comment (ISSOtm) [inline src/part2/bricks.md:15]:
```suggestion
If we instead hit the right side, we need to remove the left!
```

**Review by ISSOtm — CHANGES_REQUESTED** [review]
Nice! Something short and simple, for once :sweat_smile:

**Comment by eievui5** [comment]
Yeah, much less than I expected. Hopefully the review process won't take as long because of that :)

**Review by avivace — CHANGES_REQUESTED** [review]

**Comment by avivace** [comment]
I will say it once again: this PR needs to be rebased in order to be properly reviewed. It adds a separate file so it can even be rebased against master and we proceed from there.

**Pull request closed without merging**